src/app/examples/card-example/card-example.component.tns.html

```diff
@@ -1,15 +1,27 @@
-<StackLayout>
+<StackLayout orientation="vertical" width="100%" height="100%">
   <kirby-card>
-    <kirby-card-header [title]="'Titel'" [subtitle]="'Subtitel'">
-    </kirby-card-header>
+    <kirby-card-header [title]="'Titel'" [subtitle]="'Subtitel'"></kirby-card-header>
 
     <!-- Card content example: -->
-    <ScrollView>
-      <StackLayout class="card-content">
-        <Label text="Lorem ipsum dolor sit amet consectetur adipisicing elit. Qui quas quibusdam eligendi, debitis nemo fuga quia accusamus doloremque sit reprehenderit!" textWrap="true"></Label>        
-      </StackLayout>
-    </ScrollView>
+     <ScrollView>
+       <StackLayout class="card-content">
+         <Label text="Lorem ipsum dolor sit amet consectetur adipisicing elit. Qui quas quibusdam eligendi, debitis nemo fuga quia accusamus doloremque sit reprehenderit!" textWrap="true"></Label>
+        </StackLayout>
+      </ScrollView>
 
-    <kirby-card-footer></kirby-card-footer>
-  </kirby-card>
-</StackLayout>
+      <kirby-card-footer></kirby-card-footer>
+    </kirby-card>
+
+    <kirby-card [colortype]="'warning'">
```

Comment:
I find this a bit easier to read.

```suggestion
    <kirby-card colortype="warning">
```